webapp/components/integrations/components/installed_incoming_webhooks.jsx

```diff
@@ -111,6 +111,7 @@ export default class InstalledIncomingWebhooks extends React.Component {
                     creator={this.state.users[incomingWebhook.user_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
Instead of getting the channel here from the channel store, I think it would be better to create an index.js container for `InstalledIncomingWebhook` and use that to pull the channel. A couple reasons for that:
* `InstalledIncomingWebhooks` doesn't actually need the channel and it should only be rendering based off what is directly in the state/props (for example if the channel changed names, this wouldn't get that change because the channel is not stored in state/props)
* ChannelStore is a flux facade on top of redux (it's bypassing the redux), and we're trying to reduce the usages of it so we can eventually remove it completely

You can see an example of how something similar is done [with the AnnouncementBar component](https://github.com/mattermost/platform/tree/master/webapp/components/announcement_bar)